W-19393748-API-Manager-unregistred-information-fa (#620)
mulesoft/docs-api-manager: modules/ROOT/pages/latest-overview-concept.adoc

. The tracking registration status of each API: *Active*, *Inactive*, or *Unregistered*.  
+

> A status of *Unregistered* means that Anypoint Platform currently has no tracking data for this API version. This can occur if the endpoint has never communicated with the platform (for example, a newly declared proxy or an endpoint hosted outside an API gateway), or if the endpoint was previously active but its last activity record was purged after an extended period of inactivity (approximately 30 days). +

Reviewer: ```suggestion
A status of *Unregistered* means that Anypoint Platform currently has no tracking data for this API version. This can occur if the endpoint never communicated with the platform (for example, a newly declared proxy or an endpoint hosted outside an API gateway). It can also happen if the endpoint was active before but its last activity record was purged after an extended period of inactivity (about 30 days). +
```

. The number of contracted client applications for each API. For details,
see xref:api-contracts-landing-page.adoc[].

=== Tracking Registration Status

API Manager shows a tracking registration status for each API instance:

* *Active*: API Manager is currently receiving activity signals (for example, recent traffic via a registered gateway).
* *Inactive*: The API was previously active, but no activity has been observed for a period of time. The last active timestamp is still retained.

> * *Unregistered*: API Manager has no current tracking data for this API version. This can occur either because the endpoint has never communicated with the platform *or* because tracking data (for example, `lastActiveDate`) was purged after an extended period of inactivity (for example, ~30 days of no activity).

Reviewer: If you accept my previous suggestion, you can make the same change here.
